connectors/sources/confluence.py

```diff
@@ -232,67 +241,88 @@ def get_default_configuration(cls):
                 "options": [
                     {"label": "Confluence Cloud", "value": CONFLUENCE_CLOUD},
                     {"label": "Confluence Server", "value": CONFLUENCE_SERVER},
+                    {
+                        "label": "Confluence Data Center",
+                        "value": CONFLUENCE_DATA_CENTER,
+                    },
                 ],
                 "order": 1,
                 "type": "str",
                 "value": CONFLUENCE_SERVER,
             },
-            "username": {
+            "server_username": {
                 "depends_on": [{"field": "data_source", "value": CONFLUENCE_SERVER}],
                 "label": "Confluence Server username",
                 "order": 2,
                 "type": "str",
             },
-            "password": {
+            "server_password": {
```

Comment:
Why do we need to change these config names?

A change like this would need a migration. Otherwise, any Confluence Server user today who pulled in these changes would suddenly need to re-configure their connector, because it wouldn't have the username/password they'd previously set in the fields that the connector would now be looking at.

Reply:
Alright, so i'll revert this configuration name change. I'll just keep the change for data center.